### DISCUSSION TITLE
2007/10/19/%E6%88%91%E7%9A%84%E5%B0%8F%E8%BD%A6%E5%A5%BD%E8%84%8F%E5%95%8A

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#184] [Announcements]
# 2007/10/19/%E6%88%91%E7%9A%84%E5%B0%8F%E8%BD%A6%E5%A5%BD%E8%84%8F%E5%95%8A

我现在每天上班走的路是一条比较差的路，人车混行，路上坑坑洼洼，行人的素质也不敢恭维。

https://blog.qizhen.xyz/2007/10/19/%E6%88%91%E7%9A%84%E5%B0%8F%E8%BD%A6%E5%A5%BD%E8%84%8F%E5%95%8A

**Comment by ruanqizhen** [comment]
之前博客下的留言：

杰
2007年10月22日 19:32 编辑
你真的很强，能坚持开车，哈哈。
有异常噪音该下来检查下的，
说明你没有我爱惜车子，嘿嘿

回复
yoyo
2007年10月21日 20:21 编辑
黎明前的黑暗阿

回复
茂
2007年10月19日 09:44 编辑
等搬到新家后路就好走了：）

回复
